srcpkgs/vba-m/template

```diff
@@ -1,10 +1,9 @@
 # Template file for 'vba-m'
 pkgname=vba-m
-reverts=1292_2
```

Comment:
You can't throw out this line until `version` exceeds `1292`.

Reply:
I thought that was a commit or something, I'll add it back.